source/openapi-admin-v3.yaml

```diff
@@ -2519,7 +2519,7 @@ paths:
       parameters:
         - name: co_id
           in: query
-          description: Return only log messages associated with the given request ID.
+          description: Return only log messages associated with the given request Correlation ID.
```

Comment:
why capitalized?

Reply:
I'm sort of using it as a proper noun here, mainly because what I've read about it also uses that style.